### ISSUE TITLE
Stack pointer moving: use low byte increment instead increment of word if stack is aligned 

### ISSUE THREAD

**Issue by redirectme** [Veltas/zenv#6]
PSP and RS are (or can be done) aligned to even-address.  Therefore, when fetching the first byte from the stack, we can write "INC R" instead of "INC RR" (R is low register in register pair RR). It does not reduce the length but reduces the time (for undocumented IX\IY too).
Unfortunately it gives a small gain in zenv: PUSH and POP use  automatically 2 bytes at once, the threaded code  pointer (IY) is unaligned, RS pointer IX uses indexing instead of increment\decrement.
Here are a few places where it can be applied: 
 ```
colon_code:                       
	; Push current PC to RS   
	DEC IX                    
	DEC IXL ; from odd-addr to even-addr                       
...
two_over:
	PUSH HL
	LD HL, 4
	ADD HL, SP
	LD E, (HL)
	INC L ; from even-addr to odd-addr
	LD D, (HL)
	INC HL
	LD C, (HL)
	INC L ; from even-addr to odd-addr
...
r_from:              
	PUSH HL      
	LD L, (IX+0) 
	LD H, (IX+1) 
	INC IXL ;  from even-addr to odd-addr       
	INC IX       
	JP next      
...
exit:               
	LD C, (IX+0)
	LD B, (IX+1)
	INC IXL ;  from even-addr to odd-addr       
	INC IX      
...
to_r:             
	DEC IX                    
	DEC IXL ; from odd-addr to even-addr       
...
pick:             
	ADD HL, HL
	ADD HL, SP
	LD E, (HL)
	INC L ; from even-addr to odd-addr

```

**Comment by Veltas** [comment]
Good idea, will implement after work.

**Comment by Veltas** [comment]
Apologies for not doing "after work".

I have implemented this (7113183) using the assumption the stacks are both staying within one page, we don't need to worry about alignment and we only need to modify the low byte in every situation.
